plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.test.ts

```diff
@@ -216,9 +216,12 @@ describe('OktaGroupProvider', () => {
       };
       const provider = OktaGroupEntityProvider.fromConfig(config, {
         logger,
-        parentGroupField: 'parent_org_id',
```

Comment:
If we want to keep this, we can.. Its a week old, so it might be ok to just remove it.. Im up for debate on this.